config/default/filter.format.full_html.yml

```diff
@@ -147,3 +140,10 @@ filters:
         small__ultrawide: small__ultrawide
         small__widescreen: small__widescreen
       allowed_media_types: {  }
+  filter_iframe:
+    id: filter_iframe
+    provider: uiowa_core
+    status: false
+    weight: -40
+    settings:
+      allowed_sources: ''
```

Comment:
Wondering where this came from?

Reply:
i think filter_iframe was added after the last time we exported filter.format.full_html and that all filters are registered with the format regardless of whether the status is true or false.

Reply:
aka long overdue config update